internal/coreinternal/errorutil/http.go

```diff
@@ -13,11 +13,15 @@ func HTTPError(w http.ResponseWriter, err error) {
 	if err == nil {
 		return
 	}
+	http.Error(w, err.Error(), GetHTTPStatusCodeFromError(err))
+}
+
+func GetHTTPStatusCodeFromError(err error) int {
```

Comment:
can you please add a link to the contract as a comment to this function?

Reply:
Done